src/posts/announce-2.3.0.Final.adoc

```diff
@@ -0,0 +1,193 @@
+---
+title: Announcing Thorntail 2.3.0.Final
+publishDate: 2019-01-17
+modifyDate: 2019-01-17
+author: The Thorntail Team
+layout: post.jade
+---
+
+== Thorntail 2.3.0.Final
+
+TODO
```

Comment:
Maybe "It's a new year, time for a new Thorntail release packed with features!"

Reply:
Sounds good